software/tracksight/frontend/src/app/components/live/livegraph.tsx

```diff
@@ -33,18 +33,20 @@ const DEFAULT_LAYOUT = {
     yaxis: {},
     legend: { "orientation": "h" },
 };
-const UPDATE_INTERVAL_MS = 100; // changes update speed
```

Comment:
do we not need an update interval anymore?

Reply:
noticed it was not being used in the previous versions so I removed it but can probably readd later once we have an actual live data stream